components/button/test/button-icon.visual-diff.js

```diff
@@ -21,22 +21,25 @@ describe('d2l-button-icon', () => {
 	after(async() => await browser.close());
 
 	[
-		{ category: 'normal', tests: ['normal', 'hover', 'focus', 'disabled'] },
-		{ category: 'translucent-enabled', tests: ['normal', 'focus', 'hover'] },
+		{ category: 'normal', tests: ['normal', 'hover', 'focus', 'click', 'disabled'] },
+		{ category: 'translucent-enabled', tests: ['normal', 'hover', 'focus', 'click'] },
 		{ category: 'translucent-disabled', tests: ['normal', 'hover'] },
-		{ category: 'custom', tests: ['normal', 'hover', 'focus'] }
+		{ category: 'custom', tests: ['normal', 'hover', 'focus', 'click'] }
 	].forEach((entry) => {
 		describe(entry.category, () => {
 			entry.tests.forEach((name) => {
 				it(name, async function() {
 					const selector = `#${entry.category}`;
 
 					if (name === 'hover') await page.hover(selector);
-					else if (name === 'focus') await page.$eval(selector, (elem) => elem.focus());
+					else if (name === 'focus') await page.$eval(selector, (elem) => { elem.focus(); elem.shadowRoot.querySelector('button').classList.add('focus-visible'); });
+					else if (name === 'click') await page.$eval(selector, (elem) => elem.focus());
```

Comment:
A programatic `focus()` doesn't get the `:focus-visible` styles unless it was triggered by a keyboard event (like our menus will), so we need to manually add (and remove) the `focus-visible` class to get the "true" focus styles.